packages/hubble-idl/src/kamino.json

```diff
@@ -24,9 +24,7 @@
           "name": "pool",
           "isMut": false,
           "isSigner": false,
-          "docs": [
```

Comment:
Didnt you already update the IDL?

Reply:
yeah, I forgot to merge master, fixed now